### DISCUSSION TITLE
Stopped because an error was encountered...

### DISCUSSION THREAD

**Discussion by magne74** [Blackymas/NSPanel_HA_Blueprint#361] [Q&A] [answered]
I can't get the NSpanel blueprint to work. Get this error:
Stopped because an error was encountered at 20. februar 2023 kl. 20:18:23 (runtime: 0.27 seconds)

Template rendered invalid service: esphome.nspanel-stua_send_command_printf

Anyone that can help me?

**Comment by deejaybeam** [comment, marked as answer]
Hi
If you have copy/paste the error-message correctly, I would bet, you haven´t read the installation-instruction 100%:
you have "-" in your device-name (nspanel-stua) which causes the problem.

**Comment by magne74** [comment]
Hi Chris!

Thanks for responding:-) You were absolutely right. I had name my device nspanel_stua. Re-flashed it as nspanelstua and everything went smoothly.

Thank you!

Magne
